libs/geoservices/covid/src/lib/pages/maps/components/time-map/time-map.component.scss

```diff
@@ -40,118 +19,39 @@
       max-width: 50%;
     }
   }
-
-  #map-section-map {
-  }
-}
-
-#map-section {
-  width: auto;
-  height: 50rem;
-  display: flex;
 }
 
 #map-section h3 {
   padding: 1rem;
 }
 
-// toggle button start
-
-/* The switch - the box around the slider */
-.switch {
-  margin-right: 1rem;
-  margin-bottom: 1rem;
-  position: relative;
-  display: inline-block;
-  width: 60px;
-  height: 34px;
-}
-
-/* Hide default HTML checkbox */
-.switch input {
-  opacity: 0;
-  width: 0;
-  height: 0;
-}
-
-/* The slider */
-.slider {
-  position: absolute;
-  cursor: pointer;
-  top: 0;
-  left: 0;
-  right: 0;
-  bottom: 0;
-  background-color: #ccc;
-  -webkit-transition: .4s;
-  transition: .4s;
-}
-
-.slider:before {
-  position: absolute;
-  content: "";
-  height: 26px;
-  width: 26px;
-  left: 4px;
-  bottom: 4px;
-  background-color: white;
-  -webkit-transition: .4s;
-  transition: .4s;
-}
-
-input:checked + .slider {
-  background-color: #2196F3;
-}
-
-input:focus + .slider {
-  box-shadow: 0 0 1px #2196F3;
-}
-
-input:checked + .slider:before {
-  -webkit-transform: translateX(26px);
-  -ms-transform: translateX(26px);
-  transform: translateX(26px);
-}
-
-/* Rounded sliders */
-.slider.round {
-  border-radius: 34px;
-}
-
-.slider.round:before {
-  border-radius: 50%;
-}
-
-//toggle button end
-
 //legend start
 .legend {
-	background-color: #fff;
-	border-radius: 3px;
-	bottom: 30px;
-	box-shadow: 0 1px 2px rgba(0, 0, 0, 0.1);
-	font: 12px/20px 'Helvetica Neue', Arial, Helvetica, sans-serif;
-	padding: 10px;
-	position: absolute;
-	z-index: 1;
+  background-color: #fff;
+  border-radius: 3px;
+  bottom: 1rem;
+  box-shadow: 0 1px 2px rgba(0, 0, 0, 0.1);
+  padding: 10px;
+  position: absolute;
+  z-index: 1;
```

Comment:
Since the legend is part of the sidebar, does it make sense for it to have a drop shadow or even a background color?

If you're feeling crazy it could also be done with a flex layout. 

Sidebar: flex, column, justify space between. This would force the info and the legend on the opposite ends of the sidebar.